build.gradle

```diff
@@ -391,8 +391,8 @@ tasks.withType(Jar) {
     }
 }
 
-task wrapper(type: Wrapper) {
-    gradleVersion = '3.1'
+wrapper {
+    gradleVersion = '5.4.1'
 }
```

Comment:
@tomwhite You need to update `dockertest.gradle` as well. The needed change is in my PR here: https://github.com/broadinstitute/gatk/pull/5928/files#diff-20d46fff5956131a01d67961f7ec15c0

Reply:
Thanks @droazen, that fixes it. All the tests now pass. Are there any other changes needed before merging?

Reply:
@lbergelson suggested that we test the maven artifact uploading, as he suspects it might break in this PR. We'll test that soon and get back to you on the results.

Reply:
I fixed a deprecation warning and rebuild the wrapper itself.  Haven't changed the publishing over yet.